File: experiments_dataset_vul_detect_cwe_classification/dataset/vul_dataset/java/non-vul/code_before42.java
// From cwe-snippets, snippets_90/non-compliant/Java/cwe-0336/basic_01.java

private void func() throws Throwable
{

    SecureRandom secureRandom = new SecureRandom();

    IO.writeLine("" + secureRandom.nextInt());
    IO.writeLine("" + secureRandom.nextInt());

}

File: experiments_dataset_vul_detect_cwe_classification/dataset/vul_dataset/c/non-vul/code_before42.c
// From cwe-snippets, snippets_90/non-compliant/C/cwe-0252/char_fputc_13.c

static void func()
{
    if(GLOBAL_CONST_FIVE==5)
    {
        if (fputc((int)'A', stdout) == EOF)
        {
            printLine("fputc failed!");
        }
    }
}
